.travis/script.sh

```diff
@@ -572,7 +572,7 @@ build_samr21() {
 
 [ $BUILD_TARGET != posix-ncp ] || {
     ./bootstrap || die
-    REFERENCE_DEVICE=1 COVERAGE=1 PYTHONUNBUFFERED=1 NODE_TYPE=ncp-sim make -f examples/Makefile-posix check || die
+    CPPFLAGS=-DOPENTHREAD_CHECK_NCP_HANDLER_ENTRIES_SORTED=1 REFERENCE_DEVICE=1 COVERAGE=1 PYTHONUNBUFFERED=1 NODE_TYPE=ncp-sim make -f examples/Makefile-posix check || die
```

Comment:
> It'd be good to have travis build which enables all possible options (such that all props are included in the arrays) and then run the check and ensure the array is sorted. (The trickiness is to maintain such a travis build and ensure all features are enabled in it).

Does this build cover all features?

Reply:
This is using posix sim, which we intentionally enabled all possible features in examples/Makefile-posix. However, I just realized some properties are only enabled on RCP build, so I will turn on the check on posix-app-ncp check as well.

Reply:
I don't think `examples/Makefile-posix` enables all build features that affects the NCP dispatch table (e.g. `OPENTHREAD_CONFIG_MLE_STEERING_DATA_SET_OOB_ENABLE`, `OPENTHREAD_CONFIG_LEGACY_ENABLE`).